.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,79 @@
+name: Feature Request Form
+description: Request new or improved functionality
+title: "[FEA]: "
+labels: ["feature request"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this feature request!
+
+  - type: dropdown
+    id: new_or_improvement
+    attributes:
+      label: Is this a new feature, or an improvement?
+      options:
+        - New Feature
+        - Improvement
```

Comment:
Let's add "change" here as well. In case users want things to work differently. I.e. "it would be better if the monitor stage showed X instead of Y". Neither new nor improved